build/src/Generator/ClassFactory.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\Build\Generator;
+
+use Nette\PhpGenerator\ClassType;
+use Nette\PhpGenerator\Closure;
+use Nette\PhpGenerator\GlobalFunction;
+use Nette\PhpGenerator\Helpers;
+use Nette\PhpGenerator\Literal;
+use Nette\PhpGenerator\Method;
+use Nette\PhpGenerator\Parameter;
+use Nette\PhpGenerator\PhpNamespace;
+use Nette\PhpGenerator\Property;
+use Nette\SmartObject;
+
+/**
+ * Generate Nette class from existing source.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class ClassFactory
```

Comment:
This class is very similar to `Nette\PhpGenerator\Factory` with just a few changes. 

I wanted to include use statements and method bodies.